<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuke批渲染</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>渲染文件夹</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonOpenDir">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEditDir">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonAskDir">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>任务队列</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>文件</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>优先级</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="pushButtonRemoveOldVersion">
                  <property name="text">
                   <string>移除低版本号文件</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButtonRemove">
                  <property name="text">
                   <string>移除所选</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButtonSelectAll">
                  <property name="text">
                   <string>全选</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButtonReverseSelection">
                  <property name="text">
                   <string>反选</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>输出</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTextEdit" name="textEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="checkBoxProxy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>代理尺寸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxContinue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>出错继续渲</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPriority">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>低优先级</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>完成后</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxAfterFinish">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>什么都不做</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>关机</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>休眠</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Deadline</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>执行命令</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStop">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QToolButton" name="toolButtonOpenLog">
          <property name="text">
           <string>打开日志</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelVersion">
          <property name="text">
           <string>version</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonStart</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonStop</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonStart</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonStop</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonStop</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
